### ISSUE TITLE
Upgrade from C125 -> C126 on Win x64 via 1.68.x - Nightly (Checking C126)

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#38518]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [x] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [ ] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [x] Remove the following component folders and ensure that they're being re-downloaded after restarting the browser:
  - [x] `afalakplffnnnlkncjhbmahjfjhmlkal`: `AutoplayWhitelist.dat`, `ExtensionWhitelist.dat`, `ReferrerWhitelist.json` and `Greaselion.json`
  - [x] `CertificateRevocation`
  - [x] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
  - [x] `gkboaolpopklhgplhaaiboijnklogmbc`: Brave Ad Block List Catalog
  - [x] `iodkpdagapdfkphljnddpjlldadblomo`: Brave Ad Block Updater
  - [x] `mfddibmblmbccpadfndgakiopmmhebop`: Brave Ad Block List Catalog
  - [x] `oofiananboodjbbmdelgdommihjbkfag`: HTTPSE
  - [x] `Safe Browsing`
- [x] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers

### Upgrade

- [x] Make sure that data from the last version appears in the new version OK
- [x] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [x] With data from the last version, verify that:
  - [x] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [x] Cookies are preserved
  - [x] Installed extensions are retained and work correctly
  - [x] Opened tabs can be reloaded
  - [x] Stored passwords are preserved
  - [x] Sync chain created in previous version is retained
  - [x] Social media-blocking buttons changes are retained
  - [x] Custom filters under brave://settings/shields/filters are retained
  - [x] Custom lists under brave://settings/shields/filters are retained
  - [x] Rewards
    - [x] BAT balance is retained
    - [x] Auto-contribute list is retained
    - [x] Both Tips and Monthly Contributions are retained
    - [x] Panel transactions list is retained
    - [x] Changes to rewards settings are retained
    - [x] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [x] Ads
    - [x] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [x] Changes to ads settings are retained
    - [x] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [x] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by MadhaviSeelam** [comment]
Verification `In progress` using
```
Brave | 1.68.38 Chromium: 126.0.6478.17 (Official Build) nightly (64-bit)
-- | --
Revision | 010e3a7485e2ddf28d2510b5cc999eaa254f07f8
OS | Windows 11 Version 23H2 (Build 22631.3593)
```

<details>
<summary>C125 --> C126 on-demand upgrade check - PENDING</summary>

1. installed latest nightly `1.68.40 Chromium 125.0.6422.76 (Official Build) nightly (64-bit) ` and closed Brave
2. created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
4. set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
5. relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
6. ensured that clicking `Relaunch` restarted the browser without any issues
7. ensured Brave was updated to `1.67.59 Chromium: 125.0.6422.21 (Official Build) nightly (64-bit)` without any issues and you can restart the browser

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 5`**
----------------|----------------|----------------|---------------|------------


</details>

<details>
<summary>C125 --> C126 background upgrade check - PENDING</summary>

1. installed latest nightly `1.68.40 Chromium 125.0.6422.76 (Official Build) nightly (64-bit)` and closed Brave
2. created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
3. set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
4. started `BraveSoftwareUpdateTaskMachineUA{A215A4FF-99D2-4A6D-B2BA-2018295EB41C}` via `Task Scheduler`
5. waited ~30 min or so before launching Brave and ensured it was updated to `1.67.64 Chromium: 125.0.6422.26`  
6. restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background`

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 5`**
----------------|----------------|---------------|----------------|-----------


</details>

<details>
<summary>C126 --> C126 on-demand upgrade check - PENDING</summary>

1. installed `1.68.38 Chromium: 126.0.6478.17` via [BraveBrowserStandaloneNightlySetup_126_1_68_38.exe]`
* created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
    * `https://updates.brave.com/service/update2`
2. set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
3. relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
4. ensured that clicking `Relaunch` restarted the browser without any issues
5. ensured Brave was updated to ``  without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|---------------|----------



</details>
<details>
<summary>C126 --> C126 background upgrade check - PENDING</summary>

1. installed `1.68.38 Chromium: 126.0.6478.17` via `[BraveBrowserStandaloneNightlySetup_126_1_68_38.exe]`
2.  created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
    * `https://updates.brave.com/service/update2`
3.  set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
4. started `BraveSoftwareUpdateTaskMachineUA{9C318D44-9A78-4AFD-8B16-EC444A34098A}` via `Task Scheduler`
5. waited ~30min or so before launching Brave and ensured it was updated to ` `
6. restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background

**`step 1`** | **`step 2`** | **`step 3`** | **`step 4`** | **`step 5`** | step 6
----------------|----------------|------------|------------|---------|---------



</details>

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 10 x86` using the following build(s):

```
Brave | 1.68.39 Chromium: 126.0.6478.17 (Official Build) nightly (32-bit)
-- | --
Revision | 11f12325d90617eee0525926d4c9ac66fff6061c
OS | Windows 10 Version 22H2 (Build 19045.4412)
```

<details>
<summary>C125 --> C126 on-demand upgrade check - PASSED</summary>

* installed `1.68.36 Chromium: 125.0.6422.76` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.68.39 Chromium: 126.0.6478.17` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** 
----------------|----------------|----------------|---------------
![image](https://github.com/brave/brave-browser/assets/2602313/4f2192ce-4c64-4722-b4a7-24b64da4dc58) | ![image](https://github.com/brave/brave-browser/assets/2602313/41d0b3f5-c4b2-4635-b57e-60e2cc616e9d) | ![image](https://github.com/brave/brave-browser/assets/2602313/af91a68d-49b9-419c-95a6-248b6174f895) | ![image](https://github.com/brave/brave-browser/assets/2602313/26453047-d9d8-4791-8d26-c9c4abda7aba)\
</details>

<details>
<summary>C126 --> C126 on-demand upgrade check - PASSED</summary>

* installed `1.68.38 Chromium: 126.0.6478.17` via `BraveBrowserStandaloneNightlySetup32_126_1_68_38.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* relaunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.68.39 Chromium: 126.0.6478.17` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** 
----------------|----------------|----------------|----------------
![image](https://github.com/brave/brave-browser/assets/2602313/8928134f-01c6-43a3-9b51-ef2f98dc4592) | ![image](https://github.com/brave/brave-browser/assets/2602313/a855efcd-dd70-45d5-a247-0543de9f92fb) | ![image](https://github.com/brave/brave-browser/assets/2602313/8e342e3e-c206-4114-8827-72109f3a58c4) | ![image](https://github.com/brave/brave-browser/assets/2602313/99ec9411-e71c-4b88-94df-d85a95c6898b)

`x86` delta logs from `C:\ProgramData\BraveSoftware\Update\Log\BraveUpdate.log`:

```
[05/23/24 16:18:33.444][BraveUpdate:goopdate][1312:10632][Installing][display name: Brave Nightly][app id: {C6CB981E-DB30-4876-8639-109F8933582C}][installer path: C:\Program Files\BraveSoftware\Update\Install\{1D000B41-DB3E-4D7A-B3AA-D6AE58B51A3B}\brave_installer-delta-ia32.exe][manifest args: --chrome-sxs][installer data: ][untrusted data: ][priority: 10]
[05/23/24 16:18:33.444][BraveUpdate:goopdate][1312:10632][Running installer][C:\Program Files\BraveSoftware\Update\Install\{1D000B41-DB3E-4D7A-B3AA-D6AE58B51A3B}\brave_installer-delta-ia32.exe][--chrome-sxs][{C6CB981E-DB30-4876-8639-109F8933582C}]
[05/23/24 16:18:50.079][BraveUpdate:goopdate][1312:10632][InstallerResult][{C6CB981E-DB30-4876-8639-109F8933582C}][0]
[05/23/24 16:18:50.079][BraveUpdate:goopdate][1312:10632][InstallApp returned][0x0][{C6CB981E-DB30-4876-8639-109F8933582C}][type:1][code: 30][][]
[05/23/24 16:18:50.091][BraveUpdate:goopdate][1312:10632][Started process][5260]
[05/23/24 16:18:50.099][BraveCrashHandler][5260:4036][ConfigManager::LoadGroupPolicies][Machine is not Enterprise Managed]
[05/23/24 16:18:50.100][BraveCrashHandler][5260:4036][ConfigManager::LoadGroupPolicies][No Group Policies found under key][HKLM\Software\Policies\BraveSoftware\Update\]
[05/23/24 16:18:50.100][BraveCrashHandler][5260:4036][OmahaPolicyManager::set_policy][Group Policy][[CachedOmahaPolicy][is_initialized][0][is_managed][0][auto_update_check_period_minutes][-1][download_preference][][cache_size_limit][-1][cache_life_limit][-1][updates_suppressed][-1][-1][-1][proxy_mode][][proxy_server][][proxy_pac_url][][install_default][-1][update_default][-1]]
[05/23/24 16:18:50.102][BraveCrashHandler][5260:4036][CrashHandler::RunUntilShutdown]
[05/23/24 16:18:50.102][BraveUpdate:goopdate][1312:10632][LogEventHelper][4][12][1][[
Application update/install.
pid=1312, ver=1.3.361.149, machine=1, extern=1, private
App={C6CB981E-DB30-4876-8639-109F8933582C}, Ver=126.1.68.38, Download result=0x00000000
App={C6CB981E-DB30-4876-8639-109F8933582C}, Ver=126.1.68.38, Install result=0x00000000
]][]
[05/23/24 16:18:50.104][BraveUpdate:goopdate][10628:10620][DllEntry exit][0x00000000]
```

</details>

<details>
<summary>C125 --> C126 background upgrade check - PASSED</summary>

* installed `1.68.36 Chromium: 125.0.6422.76` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* started `BraveSoftwareUpdateTaskMachineUA{CDE49DBF-89BE-4BD4-BC13-0903509FBD4F}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.68.39 Chromium: 126.0.6478.17`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background`

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|---------------|-----------------
![image](https://github.com/brave/brave-browser/assets/2602313/154b4a75-bbe3-4f26-90e6-33adb52eb9e5) | ![image](https://github.com/brave/brave-browser/assets/2602313/22456401-3b0d-4857-9b45-5ead4aba081b) | ![image](https://github.com/brave/brave-browser/assets/2602313/ba4b2654-3385-4c4d-a5a9-90a4e0f89fdd) | ![image](https://github.com/brave/brave-browser/assets/2602313/c4d52b1d-b345-43e4-a6f9-9d53c39afe1d)


</details>

<details>
<summary>C126 --> C126 background upgrade check - PASSED</summary>

* installed `1.68.38 Chromium: 126.0.6478.17` via `BraveBrowserStandaloneNightlySetup32_126_1_68_38.exe` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* started `BraveSoftwareUpdateTaskMachineUA{0188EE31-A968-4B4D-BBB5-9A21B4FD17D8}` via `Task Scheduler`
* waited ~30min or so before launching Brave and ensured it was updated to `1.68.39 Chromium: 126.0.6478.17`
* restarted Brave several times and ensured that there's no issues re-launching Brave once it was updated via the `background

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|----------------
![image](https://github.com/brave/brave-browser/assets/2602313/d9b6cf5c-1801-4c6b-8538-05ec659b53e2) | ![image](https://github.com/brave/brave-browser/assets/2602313/8869bdfb-e3ef-451c-90f0-7b2ddb7e85b7) | ![image](https://github.com/brave/brave-browser/assets/2602313/181096e7-5f5a-4dcc-962c-de4d9631c49f) | ![image](https://github.com/brave/brave-browser/assets/2602313/f17b4502-7761-441b-8f32-da6ff4595975)

`x86` delta logs from `C:\ProgramData\BraveSoftware\Update\Log\BraveUpdate.log`:

```
[05/23/24 16:33:06.992][BraveUpdate:goopdate][3560:11312][Installing][display name: Brave Nightly][app id: {C6CB981E-DB30-4876-8639-109F8933582C}][installer path: C:\Program Files\BraveSoftware\Update\Install\{05F71260-686C-416D-89B4-66C11CD8959F}\brave_installer-delta-ia32.exe][manifest args: --chrome-sxs][installer data: ][untrusted data: ][priority: 0]
[05/23/24 16:33:06.992][BraveUpdate:goopdate][3560:11312][Running installer][C:\Program Files\BraveSoftware\Update\Install\{05F71260-686C-416D-89B4-66C11CD8959F}\brave_installer-delta-ia32.exe][--chrome-sxs][{C6CB981E-DB30-4876-8639-109F8933582C}]
[05/23/24 16:33:26.179][BraveUpdate:goopdate][3560:11312][InstallerResult][{C6CB981E-DB30-4876-8639-109F8933582C}][0]
[05/23/24 16:33:26.179][BraveUpdate:goopdate][3560:11312][InstallApp returned][0x0][{C6CB981E-DB30-4876-8639-109F8933582C}][type:1][code: 2][]["C:\Program Files\BraveSoftware\Brave-Browser-Nightly\Application\brave.exe" --from-installer]
[05/23/24 16:33:26.179][BraveUpdate:goopdate][3560:11312][Started process][6120]
[05/23/24 16:33:26.179][BraveUpdate:goopdate][3560:11312][LogEventHelper][4][12][1][[
Application update/install.
pid=3560, ver=1.3.361.149, machine=1, extern=1, private
App={C6CB981E-DB30-4876-8639-109F8933582C}, Ver=126.1.68.38, Download result=0x00000000
App={C6CB981E-DB30-4876-8639-109F8933582C}, Ver=126.1.68.38, Install result=0x00000000
]][]
[05/23/24 16:33:26.195][BraveCrashHandler][6120:5464][ConfigManager::LoadGroupPolicies][Machine is not Enterprise Managed]
[05/23/24 16:33:26.195][BraveCrashHandler][6120:5464][ConfigManager::LoadGroupPolicies][No Group Policies found under key][HKLM\Software\Policies\BraveSoftware\Update\]
[05/23/24 16:33:26.195][BraveCrashHandler][6120:5464][OmahaPolicyManager::set_policy][Group Policy][[CachedOmahaPolicy][is_initialized][0][is_managed][0][auto_update_check_period_minutes][-1][download_preference][][cache_size_limit][-1][cache_life_limit][-1][updates_suppressed][-1][-1][-1][proxy_mode][][proxy_server][][proxy_pac_url][][install_default][-1][update_default][-1]]
[05/23/24 16:33:26.195][BraveCrashHandler][6120:5464][CrashHandler::RunUntilShutdown]
[05/23/24 16:33:26.273][BraveUpdate:goopdate][1608:2292][DllEntry exit][0x00000000]
```
</details>
